doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -700,7 +706,9 @@ OSSL_CMP_certConf_cb() returns I<fail_info> if it is not equal to 0,
 else 0 on successful validation,
 or else a bit field with the B<OSSL_CMP_PKIFAILUREINFO_incorrectData> bit set.
 
-All other functions return 1 on success, 0 on error.
+All other functions, including OSSL_CMP_CTX_reinit()
```

Comment:
I'm not sure the _including..._ phrase needs to be here.  It's kind of implicit in the "all other" part.

Reply:
Right, the "including ..." part is implied by the rest, but I thought it could be helpful to explicitly mention this new function here, for instance for the case that a reader greps on the man page to quickly find the result behavior of that function.
Thanks @paulidale for giving the approval regardless of this point.